src/bloomfilter/lib.rs

```diff
@@ -17,7 +17,7 @@ extern crate bit_vec;
 extern crate rand;
 extern crate siphasher;
 
-use bit_vec::BitVec;
+pub use bit_vec::BitVec;
```

Comment:
This allows a user to type `use boomfilter::{Bloom, BitVec}` and `Bloom::from_existing(BitVec::from_bytes(...), ...)` without having to depend on BitVec themselves (though they can depend on it if they need more control over the BitVec construction)